cmd/tracee/cmd/analyze.go

```diff
@@ -106,17 +107,41 @@ func command(cmd *cobra.Command, args []string) {
 
 	// Set up printer output (outpath:format)
 	outputArg := viper.GetString("output")
+
+	// default for legacy
+	p, err := printer.New(config.PrinterConfig{
+		OutFile: os.Stdout,
+		Kind:    "ignore",
+	})
+
+	if err != nil {
+		logger.Fatalw("Failed to initialize default printer")
+	}
+
+	isLegacy := false
```

Comment:
No, the comment is just not well formed. I'll rewrite it. The idea was just that I don't want to check for nil inside the usecase file in case we use legacy format (Unfortunately implementing legacy output as a printer would be bending over backwards just for this feature, as it would involve finding->event->finding conversion).